components/ActionBar/index.js

```diff
@@ -206,24 +206,22 @@ const ActionBar = ({
     },
     {
       title: t('article/actionbar/userprogress'),
-      element:
-        document && document.userProgress && displayMinutes > 1 ? (
-          <UserProgress
-            documentId={document.id}
-            forceShortLabel={forceShortLabel}
-            userProgress={document.userProgress}
-            noCallout={
-              mode === 'articleOverlay' ||
-              mode === 'bookmark' ||
-              mode === 'seriesEpisode'
-            }
-            noScroll={mode === 'feed'}
-          />
-        ) : (
-          <></>
-        ),
+      element: (
+        <UserProgress
+          documentId={document.id}
+          forceShortLabel={forceShortLabel}
+          userProgress={document.userProgress}
```

Comment:
What was the incentive behind splitting the show/hide logic the component itself and and the `show`attribute?

Reply:
the ActionBar component doesnt use the `withProgressApi`, which provides `showConsentPrompt`